docs/UserGuide.md

```diff
@@ -345,22 +321,104 @@ Examples:
+
 
 Examples:
-* `c-tag-find friday` returns all employees whose tags contain `friday`.
-* `c-tag-find friday parttime` returns all employees whose tags contain `friday`, `parttime`.<br>
+* Let's say you want to find out who are the available to work on `Monday` or `Friday`. In this case, the two key
+ words are `Monday` and `Friday`. You can follow these steps.
+
+Steps to find employees who are available to work on `Monday` or `Friday`:
+1. Type `c-tag-find monday friday` into the _Command Box_.
+2. Press `Enter` to execute.
+
+Outcome:
+1. The _Result Display_ will show the total number of employees who are can work on `Friday` or `Monday`.
+
+2. Employees who are can work on `Friday` or `Monday` will be listed in the _Employee Directory_ pane.
+
+{Screenshot}
+
+#### 3.6 Locating all employees who are available to work today: `c-today`
+
+Finds employees whose tags contain today's day (i.e. `Wednesday`, `Tuesday`, etc).
+
+Format: `c-today`
+
+* The search is case-insensitive (e.g `Friday` will match `friday`).<br>
+* The application will automatically get today's day based on the current computer's date.<br>
+* Only full words will be marched (e.g `Fri` will not match `Friday`).<br>
+
+**:information_source: Note:**<br>
+* In tCheck, if an employee contains a tag for a specific day, it means this employee is available on that day
+. You may set an employee's tags when adding the employee into tCheck (Refer to `c-add` command), or edit the
+tag(s) by using editing employee's information feature (Refer to `c-edit` command).<br>
+
+Examples:
+* Let's say today is `Thursday`. You want to find out who are the available manpower for today. You can follow these
+ steps.
+
+Steps to find today's available manpower:
+1. Type `c-today` into the _Command Box_.
+2. Press `Enter` to execute.
+
+Outcome:
+1. The _Result Display_ will show 3 information:
+    
+    a. Today's day. (e.g. Today is THURSDAY.)
+    
+    b. Total number of employees who are available today.
+    
+    c. The success message.
+2. The available manpower for today will be listed in the _Employee Directory_ pane.
+
+{Screenshot}
+
+#### 3.7 Locating all employees who are available to work tomorrow: `c-tomorrow`
+
+Finds employees whose tags contain tomorrow's day (i.e. `Wednesday`, `Tuesday`, etc).
+
+Format: `c-tomorrow`
+
+* The search is case-insensitive. e.g `Friday` will match `friday`.<br>
+* The application will automatically get tomorrow's day based on the current computer's date.<br>
+* Only full words will be marched (e.g `Fri` will not match `Friday`).<br>
+
+**:information_source: Note:**<br>
+* In tCheck, if an employee contains a tag for a specific day, it means this employee is available on that day
+. You may set an employee's tags when adding the employee into tCheck (Refer to `c-add` command), or edit the
+tag(s) by using editing employee's information feature (Refer to `c-edit` command).<br>
```

Comment:
Should this be placed in between the  div line to show the blue box? Since this is additional information about the tags

Reply:
Thanks for pointing out! Will edit the div lines. Thanks